source/MRViewer/MRViewportCamera.cpp

```diff
@@ -389,6 +399,11 @@ void Viewport::preciseFitDataToScreenBorder( const FitDataParams& fitParams )
     params_.objectScale = dif.length();
     if ( params_.objectScale == 0.0f )
         params_.objectScale = 1.0f;
+    if ( params_.objectScale > cMaxObjectScale )
```

Comment:
The right part is squared scale and the left part?

Reply:
Introduced new constant to avoid confusion.